src/actions/timeEntries.js

```diff
@@ -73,7 +73,7 @@ export const editTimeEntry = (timeEntryId, timeEntry, oldDateOfWork) => {
     try {
       const res = await axios.put(url, timeEntry);
       dispatch(updateTimeEntries(timeEntry, oldDateOfWork));
-      return res.status;
+      return res.status
```

Comment:
Did you delete the semicolon on purpose?

Reply:
Hmm, no, maybe I accidentally deleted it.